src/openrct2-ui/windows/Ride.cpp

```diff
@@ -1048,6 +1048,7 @@ static void window_ride_draw_tab_vehicle(rct_drawpixelinfo* dpi, rct_window* w)
         if (rideEntry == nullptr)
             return;
 
+        const auto& rtd = ride->GetRideTypeDescriptor();
```

Comment:
Move this closer to the if below (or do it directly on the if to reduce the scope of the variable)